main.py

```diff
@@ -45,19 +52,19 @@ async def files(message: types.Message):
             ext_count += 1
     if ext_count == 0:
         await message.react([react])
-        await bot.send_message(-1002027282215, message.from_user.username, message_thread_id=45)
-        await bot.send_document(-1002027282215, document=message.document.file_id, message_thread_id=45)
+        await bot.send_message(ARCHIVE, message.from_user.username, message_thread_id=FILES)
+        await bot.send_document(ARCHIVE, document=message.document.file_id, message_thread_id=FILES)
 
 
-# пересылает мемы из указанных групп
-@dp.message((F.chat.id == -1001675679569) & (F.forward_origin.chat.id.in_({-1001595506698, -1001081170974, -1001009232144, -1001399874898})))
+# forwards memes from the specified groups
+@dp.message((F.chat.id == ORIGIN) & (F.forward_origin.chat.id.in_({-1001595506698, -1001081170974, -1001009232144, -1001399874898})))
```

Comment:
memes list go in a separate config file?

Reply:
I don't see much point, these are public groups. In any case, this rule will be changed soon. We plan to build normal rules on the data that the bot will collect.